webapp/src/Service/ImportExportService.php

```diff
@@ -854,7 +854,7 @@ public function importTeamsJson(array $data, ?string &$message = null, ?array &$
                     'name' => @$team['name'],
                     'display_name' => @$team['display_name'],
                     'publicdescription' => $team['public_description'] ?? @$team['members'],
-                    'room' => @$team['room'],
+                    'location' => @$team['location']['description'],
```

Comment:
Is there a way how we can now just if unset pick the `x,y,rot` and put it as a string?

Reply:
Usually we use the seat number (such as B05) for the `location` here, but x, y, rot in the ccs spec represent distance and rotation angle. I think they do not correspond well directly.

Reply:
You can already use the `description` field, so for your usecase you would put the seat number there (or would you still need more?)

I'm suggesting the situation where that field is not set and the spec is followed. During the ICPC WF we would get the information in DOMjudge without much extra effort so we know where a team would be located in case of problems.

Reply:
But then if you import + export you get something else, while it would be nice if it's idempotent?

Reply:
> You can already use the `description` field, so for your usecase you would put the seat number there (or would you still need more?)

AFAIK, the `room` is formally used in built-in printing and balloon.